modules/virt-about-services.adoc

```diff
@@ -13,7 +13,11 @@ ClusterIP:: Exposes the service on an internal IP address within the cluster. `C
 NodePort:: Exposes the service on the same port of each selected node in the cluster. `NodePort` makes a service accessible from outside the cluster.
 
 LoadBalancer:: Creates an external load balancer in the current cloud (if supported) and assigns a fixed, external IP address to the service.
-
++
+[NOTE]
+====
```

Comment:
> See the _Additional resources_ section for more information.

I can see why you included this line, but it assumes that this module is always going to be in this assembly (or that any assembly that it's in will have the right links in the add'l resources section). 

Suggestion to take or leave: I might split this module into two modules - with the dual-stack support as its own module - so that you can put the links right after this note in the assembly.